plugin/src/main/resources/io/jenkins/plugins/casc/Attribute/schema.jelly

```diff
@@ -28,20 +28,16 @@
         "type": "boolean"
       </j:case>
       <j:case value="int">
-        "type": "integer",
-        "format": "uint32"
```

Comment:
pajv wasn't supporting this, its not built into json-schema but you are allowed to extend, seems unnecessary to add the additional formats imo